src/durableorchestrationclient.ts

```diff
@@ -0,0 +1,443 @@
+import { isURL } from "validator";
+import { Constants, DurableOrchestrationStatus, HttpManagementPayload, IFunctionContext, IHttpRequest,
+    IHttpResponse, IRequest, OrchestrationClientInputData, OrchestrationRuntimeStatus, Utils, WebhookClient,
+} from "./classes";
+
+/**
+ * Returns an OrchestrationClient instance.
+ * @param context The context object of the Azure function whose body
+ *  calls this method.
+ * @example Get an orchestration client instance
+ * ```javascript
+ * const df = require("durable-functions");
+ *
+ * module.exports = df.orchestrator(function*(context) {
+ *     const client = df.getClient(context);
+ *     const instanceId = await client.startNew(req.params.functionName, undefined, req.body);
+ *
+ *     return client.createCheckStatusResponse(instanceId);
+ * });
+ * ```
+ */
+export function getClient(context: unknown): DurableOrchestrationClient {
```

Comment:
Mostly to enable unit tests like [this one,](https://github.com/Azure/azure-functions-durable-js/blob/3151cfd85b67e1a29521788ee6d24423fe471458/test/unit/orchestrationclient-spec.ts#L68) which test wrongly-shaped objects getting passed in but otherwise throw compilation errors (even with explicit type assertions.)

TypeScript type-checking only happens at compile-time; by design types are stripped out during transpilation, and no type-checking happens at runtime, so defining `context's type as `unknown` versus `IFunctionContext` has (barring some corner case or nuance I don't yet understand) no effect on how the code runs, but does allow for testing possible scenarios.